src/styles/index.scss

```diff
@@ -2,2 +1,237 @@
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-inline: 50px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+}
+
+.logo {
+  display: flex;
```

Comment:
The `display: flex;` property for the .logo class seems unnecessary because there is no child element in the .logo class to align.